ui/imports/Constants.qml

```diff
@@ -119,4 +119,10 @@ QtObject {
     readonly property int browserEthereumExplorerBlockchair: 3
 
     readonly property int repeatHeaderInterval: 2
+
+    readonly property int maxUploadFiles: 5
+    readonly property var maxUploadFilesizeMB: 0.5
+
+    readonly property var acceptedImageExtensions: [".png", ".jpg", ".jpeg", ".svg", ".gif"]
+    readonly property var acceptedDragNDropImageExtensions: [".png", ".jpg", ".jpeg", ".heif", "tif", ".tiff"]
```

Comment:
Does it make sense to allow file types for drag and drop that are *not* allowed as general image extensions?
In otherwords, shouldn't these two lists be the same?

Reply:
Good point, but what happens if we wanted to be able to drag and drop something that wasn't allowed to be sent in